Mathlib/Order/WellFounded.lean

```diff
@@ -286,3 +286,27 @@ theorem WellFounded.induction_bot {α} {r : α → α → Prop} (hwf : WellFound
 #align well_founded.induction_bot WellFounded.induction_bot
 
 end Induction
+
+section WellFoundedLT
+
+variable [Preorder α] [Preorder β] {f : α → β}
+
+theorem WellFoundedLT.of_strictMono [WellFoundedLT β] (hf : StrictMono f) : WellFoundedLT α where
```

Comment:
I would actually name this different to unlock dot notation. Same for the others in this file.
```suggestion
theorem StrictMono.wellFoundedLT [WellFoundedLT β] (hf : StrictMono f) : WellFoundedLT α where
```